README.rst

```diff
@@ -951,16 +951,21 @@ among the most popular alternatives:
   <https://pypi.org/project/devpi/>`_.
   (version: 2.1.4, access date: 8/3/2015)
 
+- Check this SO question: `How to roll my own pypi
+  <http://stackoverflow.com/questions/1235331/how-to-roll-my-own-pypi>`_
+
+
+Unmaintained or archived
+------------------------
+
+These projects were once alternatives to pypiserver but are now either unmaintained or archived.
+
 - `pip2pi <https://github.com/wolever/pip2pi>`_
   a simple cmd-line tool that builds a PyPI-compatible local folder from pip requirements
-  (version: 0.6.7, access date: 8/3/2015)
```

Comment:
Good that this version is removed but I am not sure why do you consider this project unmaintained, it is not very active indeed but still the last commit was about a year ago and the project is not archived. Maybe I have not found something in the issues or other places, could you correct me if I am wrong?

Reply:
Yeah, this totally depends on your definition of unmaintained. It has no commits in about a year, no release more than 1.5 years, and unanswered issues going back to 2015. That is unmaintained in my book, but I understand if you want to be more conservative in your readme. On the other hand, I wouldn't like to be the one who forwards others to something that is this stale.

Reply:
Fair enough, that's indeed very subjective, you've got a point here. :ok_hand: don't mean to offend the owners of the other project but that's fair enough, they can poke us back :)